tensorflow_2/README.md

```diff
@@ -0,0 +1,45 @@
+# Samples for Notebook to Kubeflow Deployment using TensorFlow 2.0 Keras
+This directory aims at building Kubeflow use case samples with Tensorflow 2.0 Keras training code demonstrating 'Customer User Journey' (CUJ) in the process.
```

Comment:
Add link to CUJ

Reply:
Can you please elaborate on what you mean by 'link to CUJ'? Do you want me to put a link to a documentation which explains 'CUJ'?

Reply:
I thought there would be a slide that demonstrates the CUJ but don't worry about it if not.